app/blueprints/fund_builder/forms/templates.py

```diff
@@ -2,17 +2,19 @@
 from wtforms import FileField
 from wtforms import HiddenField
 from wtforms import StringField
-from wtforms.validators import DataRequired, Regexp
+from wtforms.validators import DataRequired
+from wtforms.validators import Regexp
 
 
 class TemplateUploadForm(FlaskForm):
     template_name = StringField("Template Name", validators=[DataRequired()])
     file = FileField("Upload File", validators=[DataRequired()])
 
+
```

Comment:
Shall we remove these changes some time because of these it takes us to merge conflict issues so better only have mandatory changes with the PR please check the full pr if some things is related to formatting please remove

Reply:
@nuwan-samarasinghe  this is from pre-commit hooks change